src/components/screens/consul/Proposals/NewProposal.js

```diff
@@ -302,10 +302,9 @@ const INPUTS = [
     autoCompleteType: 'off',
     autoCapitalize: 'none',
     rules: {
-      required: true,
+      required: texts.consul.startNew.leerError,
```

Comment:
can you please refactor all `leerError`s to some English equivalent?

Reply:
I will do. 👍

Reply:
eddf2ad3a7cb0b2c717a22027b3979abd3e0388c